certified-connectors/Alkymi/readme.md

```diff
@@ -0,0 +1,61 @@
+##Alkymi Connector
+
+Alkymi provides an external API which allows you to easily access the data extracted from unstructured documents 
+processed by the Alkymi app. This connector will allow you to easily pipe data from the external API to any of the apps 
+in the Microsoft PowerApps ecosystem.
+
+##Prerequisites
+
+You will need to have an active `alkymi.io` domain and API key to leverage this connector. 
+
+https://github.com/microsoft/PowerPlatformConnectors.git
+##Supported Operations
+####Triggers
```

Comment:
@tori-alkymi Please fix the formatting of the readme.md file by referring to the template.

Reply:
@tori-alkymi Can you please let us know why this comment was resolved?